### ISSUE TITLE
After adding the serverless-resources-env plugin, I get an error about "missing region in config"

### ISSUE THREAD

**Issue by roboweaver** [rurri/serverless-resources-env#2]
Serverless: Packaging service...
Serverless: Uploading CloudFormation file to S3...
Serverless: Uploading service .zip file to S3 (6.28 MB)...
Serverless: Updating Stack...
Serverless: Checking Stack update progress...
....................................
Serverless: Stack update finished...
Serverless: Removing old service versions...
Service Information
service: walkmeApi
stage: dev
region: us-west-2
api keys:
  None
endpoints:
  POST - https://bdh02rome8.execute-api.us-west-2.amazonaws.com/dev/users
  GET - https://bdh02rome8.execute-api.us-west-2.amazonaws.com/dev/users
  GET - https://bdh02rome8.execute-api.us-west-2.amazonaws.com/dev/users/{id}
  PUT - https://bdh02rome8.execute-api.us-west-2.amazonaws.com/dev/users/{id}
  DELETE - https://bdh02rome8.execute-api.us-west-2.amazonaws.com/dev/users/{id}
functions:
  walkmeApi-dev-update: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-update
  walkmeApi-dev-triggerEmail: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-triggerEmail
  walkmeApi-dev-triggerAlert: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-triggerAlert
  walkmeApi-dev-delete: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-delete
  walkmeApi-dev-triggerCreate: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-triggerCreate
  walkmeApi-dev-get: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-get
  walkmeApi-dev-list: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-list
  walkmeApi-dev-create: arn:aws:lambda:us-west-2:235544613716:function:walkmeApi-dev-create
Serverless: [serverless-resources-env] Looking up resources for CF Named: walkmeApi-dev
 
  Config Error -------------------------------------------
 
     Missing region in config
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
 
  Stack Trace --------------------------------------------
 
ConfigError: Missing region in config
    at Request.VALIDATE_REGION (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/event_listeners.js:81:45)
    at Request.callListeners (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at callNextListener (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/sequential_executor.js:95:12)
    at /usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/event_listeners.js:75:9
    at finish (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/config.js:308:7)
    at /usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/config.js:324:9
    at SharedIniFileCredentials.get (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/credentials.js:126:7)
    at getAsyncCredentials (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/config.js:318:24)
    at Config.getCredentials (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/config.js:338:9)
    at Request.VALIDATE_CREDENTIALS (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/event_listeners.js:70:26)
    at Request.callListeners (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/sequential_executor.js:101:18)
    at Request.emit (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
    at Request.emit (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/request.js:668:14)
    at Request.transition (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at Request.runTo (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/request.js:394:15)
    at Request.send (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/request.js:358:10)
    at makeRequest (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/service.js:193:27)
    at svc.(anonymous function) [as describeStackResources] (/usr/local/lib/node_modules/serverless/node_modules/aws-sdk/lib/service.js:451:23)
    at tryCatcher (/Users/robweaver/Sites/workfront/SharedDeployScripts/AWS/serverless/WalkMe/node_modules/bluebird/js/release/util.js:16:23)
    at ret (eval at makeNodePromisifiedEval (/Users/robweaver/Sites/workfront/SharedDeployScripts/AWS/serverless/WalkMe/node_modules/bluebird/js/release/promisify.js:184:12), <anonymous>:13:39)
    at ServerlessResourcesEnv.fetchCFResources (/Users/robweaver/Sites/workfront/SharedDeployScripts/AWS/serverless/WalkMe/node_modules/serverless-resources-env/index.js:106:99)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
 
     Please report this error. We think it might be a bug.
 
  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       6.7.0
     Serverless Version: 1.4.0

**Comment by rurri** [comment]
Thanks for reporting this. Will work on reproducing. In the meantime, you could try setting the region in  your .yml file under provider:

```
provider:
  name: aws
  runtime: nodejs4.3
  region: us-west-2
```

If that does not work you can also set an environment variable in your work environment:
`AWS_DEFAULT_REGION=us-west-2`

**Comment by rurri** [comment]
This is now fixed in the master branch. You can pull from master for it to work. I also confirmed that setting the region in the environment variable `AWS_DEFAULT_REGION=us-west-2` also works.

AWS_DEFAULT_REGION=us-west-2

https://github.com/rurri/serverless-resources-env/commit/07f9a035d8f3a667e5b19bc219f447323e046281

**Comment by rurri** [comment]
Fix released as part of 0.3.0
